docs/_edot-sdks/java/setup/index.md

```diff
@@ -7,7 +7,52 @@ parent: EDOT Java
 
 # Setting up the EDOT Java Agent
 
-TODO:
-- where to download
-- explicit description of basic setup here (even if it overlaps with upstream docs)
-- link to upstream docs for more advanced setup use cases
\ No newline at end of file
+## Download
+
+Latest release: [![Maven Central](https://img.shields.io/maven-central/v/co.elastic.otel/elastic-otel-javaagent?label=elastic-otel-javaagent)](https://mvnrepository.com/artifact/co.elastic.otel/elastic-otel-javaagent/latest)
+
+Latest snapshot: [![Sonatype Nexus](https://img.shields.io/nexus/s/co.elastic.otel/elastic-otel-javaagent?server=https%3A%2F%2Foss.sonatype.org&label=elastic-otel-javaagent)](https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=co.elastic.otel&a=elastic-otel-javaagent&v=LATEST)
+
+## Prerequisites
+
+In order to send your data to Elastic using EDOT Java, you need to have the following prerequisites
+- an [Elastic](/quickstart/) deployment
+- an [EDOT collector](/edot-collector/) deployment
+
+TODO: how to create an API key for the OTEL SDK ? Should probably be distinct from the collector API key.
+
+## Run
+
+Use the `-javaagent:` JVM argument with the path to agent jar, this requires to modify the JVM arguments and restart
+the application.
+
+```bash
+java \
+  -javaagent:/path/to/agent.jar \
+  -jar myapp.jar
+```
+
+For applications deployed with Kubernetes, we recommend using [OpenTelemetry Operator](./k8s).
+
+## Configuration
```

Comment:
[for reviewer] here we call this section 'documentation' but this is more a 'minimal documentation' with the bare minimum people should have to configure in practice. It should be almost identical for all SDKs like the prerequisites though.